src/core.js

```diff
@@ -425,6 +425,16 @@ jQuery.extend({
 	support: support
 });
 
+// jsHint would error on this code due to the Symbol not being defined in ES5.
```

Comment:
@mzgol fwiw, node has `Symbol` as of `v0.12`, and all `io.js` versions have it as well